### PR TITLE
Update MATURITY for C30, RE50, CO20, CO40, CO50, CS10, IN10

### PR DESCRIPTION
**Pull request by ulysses-you**
### _Why are the changes needed?_

For now, we have 6 different release managers and 8 new committers.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by ulysses-you** [comment]
cc @yaooqinn

**Comment by yaooqinn** [comment]
can we also update C30, CO50, CS10?

**Comment by yaooqinn** [comment]
please also add the `id`s in the pr title

File: MATURITY.md [hunk 1]
```diff
@@ -74,27 +74,27 @@ Mentors and community members are encouraged to contribute to this page and comm
...
+|------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CS10 | The project maintains a public list of its contributors who have decision power. The project's PMC (Project Management Committee) consists of those contributors.                                                         | :white_check_mark:~Committers and PPMC members are listed on the website’s community page, see [newcommitter](https://community.apache.org/newcommitter.html)                                                    |
```
Comment (yaooqinn) [inline MATURITY.md:88]: https://community.apache.org/newcommitter.html I don't see our committers listed there

Comment (yaooqinn) [inline MATURITY.md:88]: please also remove ~

File: MATURITY.md [hunk 2]
```diff
@@ -74,27 +74,27 @@ Mentors and community members are encouraged to contribute to this page and comm
...
+| CO40 | The community strives to be meritocratic and gives more rights and responsibilities to contributors who, over time, add value to the project.                          | :white_large_square:. The community has elected 0 new PPMC members and 8 new committers during incubation, based on meritocracy.                                                                                                      |
+| CO50 | The project documents how contributors can earn more rights such as commit access or decision power, and applies these principles consistently.                        | :white_large_square: The process to become a committer or PPMC member is documented on the website, see [newcommitter](https://community.apache.org/newcommitter.html)                                                                |
```
Comment (yaooqinn) [inline MATURITY.md:80]: ditto

File: MATURITY.md (hunk 1 above)
Comment (ulysses-you) [reply MATURITY.md]: good catch

**Review by yaooqinn — APPROVED** [review]

**Comment by ulysses-you** [comment]
thank you @yaooqinn , merging to master

**Pull request closed without merging**